src/isa/riscv64/instr/rvc/exec.h

```diff
@@ -67,6 +67,18 @@ def_EHelper(c_jalr) {
 #endif
 }
 
+def_EHelper(c_ebreak) {
+#ifdef CONFIG_RV_DEBUG
+  rtl_trap(s, s->pc, 3);
+  rtl_mv(s, s0, t0);
```

Comment:
why `rtl_mv(s, s0, t0);` and `rtl_priv_jr(s, s0);`?

From isa/riscv64/instr/priv/system.c, I see:

```c
#ifdef CONFIG_RV_DEBUG
    } else if (id == 1) { // ebreak
      rtl_trap(s, s->pc, 3);
      rtl_mv(s, jpc, t0);
#elif defined(CONFIG_EBREAK_AS_TRAP)
...
#endif
```

So should we only do the `rtl_mv(s, jpc, t0);` here?

Reply:
`rtl_priv_jr(s, s0);` is executed by ebreak in src/isa/riscv64/instr/priv/exec.h:21, I think c_ebreak should do the same.

Reply:
I see. Then should we move ` rtl_priv_jr(s, s0);` to `#ifdef CONFIG_RV_DEBUG`? This line isunused in CONFIG_EBBREAK_AS_TRAP which contains a longjmp

Reply:
Yes, this is more intuitive in code, I will fix it.